src/client.js

```diff
@@ -445,18 +444,19 @@ class Client extends EventEmitter {
     }
 
     customMessage(postData, channelID) {
+        const preparedPostData = { ...postData };
```

Comment:
Not sure why this is needed here.

Reply:
Please see the comment below, the main case why I did this is enabled eslint rule `no-param-reassign`

Reply:
I've chosen a different solution in the new PR. This way is a bit irritating. Destructuring is not needed here.